op-supervisor/supervisor/backend/processors/executing_message.go

```diff
@@ -13,7 +13,7 @@ import (
 
 type EventDecoderFn func(*ethTypes.Log, depset.ChainIndexFromID) (*types.ExecutingMessage, error)
 
-func DecodeExecutingMessageLog(l *ethTypes.Log, depSet depset.ChainIndexFromID) (*types.ExecutingMessage, error) {
+func MessageFromLog(l *ethTypes.Log) (*types.Message, error) {
```

Comment:
Nit: I would use the _executing_ terminology either in the fn name or in the godoc.